fasta/multiFa.go

```diff
@@ -22,6 +23,19 @@ func RefPosToAlnPos(record Fasta, RefPos int) int {
 	return AlnPos
 }
 
+//RefPosToAlnPosCounter is like RefPosToAlnPos, but the starting AlnPos and refCounter can be set.
+func RefPosToAlnPosCounter(record Fasta, RefPos int, refStart int, alnStart int) int {
+	for t := alnStart; refStart < RefPos; t++ {
```

Comment:
Do we need to check the input to make sure that refStart is less than or equal to RefPos?  If not, we could set both refStart and alnStart to zero.